src/sentinel.c

```diff
@@ -1212,6 +1212,10 @@ int sentinelUpdateSentinelAddressInAllMasters(sentinelRedisInstance *ri) {
         if (match->link->pc != NULL)
             instanceLinkCloseConnection(match->link,match->link->pc);
 
+	/*Remove any sentinel with port number set to 0 */
```

Comment:
```suggestion
	/* Remove any sentinel with port number set to 0 */
```

Something weird with indentation - is it a tab perhaps?